app/models/external_link.rb

```diff
@@ -61,5 +61,9 @@ def self.edit_action; end
   def self.index_action; end
   def self.show_url(id); end
   def self.show_link_args(id); end
+  def self.external_link_path(*args); end
```

Comment:
Define ExternalLink#external_link_path methods
Bootstrap4 uses `x_path` helpers in place of `show_link_args`.
ExternalLink has its own definition of `show_link_args`.
So **should ExternalLink have its own, corresponding definition of `external_link_path`**?

Reply:
I believe this stretch of definitions is just defining these methods as "nil" for an external link, basically. I don't believe we ever call show_link_args (or would ever call object_path) on an external link, so it probably does not need its own path method. The only place i could think of that happening is in rss_logs, but i don't believe they get logged that way either.

Reply:
I'll ask @pellaea to glance at these two lines of code; I think he wrote ExternalLink.

Reply:
As you've seen from the email, he doesn't remember enough to comment.
All these empty methods were added 4 years ago by @pellaea. The commit message:
> "Oops! Forgot to tell API how to render external links!"
I will merge the PR **with** the new methods. My thought: 
If presence/absence of those methods causes a problem, it's easier to figure out and debug if the methods are present than if they are absent.
But feel free to remove them fi you think they are a problem.